test-network-function/tnf_config.yml

```diff
@@ -0,0 +1,43 @@
+generic:
+  containersUnderTest:
+    - namespace: default
+      podName: test
+      containerName: test
+      defaultNetworkDevice: eth0
+      multusIpAddresses:
+        - 10.217.0.8
+    - namespace: default
+      podName: partner
+      containerName: partner
+      defaultNetworkDevice: eth0
+      multusIpAddresses:
+        - 10.217.0.29
+  partnerContainers:
+    - namespace: default
+      podName: partner
+      containerName: partner
+      defaultNetworkDevice: eth0
+      multusIpAddresses:
+        - 10.217.0.29
```

Comment:
We need to take this into account:
https://github.com/test-network-function/test-network-function/pull/128
Particularly:
https://github.com/test-network-function/test-network-function/blob/main/test-network-function/generic_test_configuration.yml#L19-L22

Reply:
Missed that with rebasing, should be sorted now.